styled/hamburger.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components/native';
+import { TouchableOpacity } from 'react-native';
+
+//TODO: Fix top to match figma once we figure out iPhone alignment
+export const HamburgerButton = styled(TouchableOpacity)`
+  background-color: #fff;
+  width: 40px;
+  height: 40px;
+  zindex: 100;
+  position: absolute;
+  top: 63px;
+  left: 16px;
```

Comment:
I think you can solve your positioning issues by moving <Hamburger navigation={this.props.navigation} /> into the MapView in `MapScreen.js`, right above SearchBar. Then you can make the top here the same as the top margin on SearchBar (20px I believe), and it should stay lined up with the height of the search bar, which has mostly been looking fine on different kinds of screens.